Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
+    private StringBuilder logString;
+    public string Label { get; set; }
+
+    private LogType logType;
+    private bool logCommonColumns;
+
+    private int nbLines;
+    private bool createStringOverTime;
+    private StringBuilder currentLineLogged;
+    private SortedDictionary<string, string> currentLogs;
```

Comment:
Whats the difference between `logs` and `currentLogs`  dictionaries ? - not sure I understand the difference just from the naming convention alone. And it seems like logs are added to both dictionaries in e.g. `Add()` ?

EDIT: I understand now. I think you could maybe call it `currentRow` or `currentLogRow` to make it clear its relationship also to your function `TerminateRow`. (which we could also call `EndRow` i guess)

Reply:
The `logs` dictionary contains all the logs of each row while the `currentsLogs` dictionary contains only the logs of the current line.
I think we don't really need to rename `logs` because this is just the dictionary where all the logs are stored, what do you think ?
I agree with your second proposal, let's rename `currentLogs` to `currentLogRow` and `TerminateRow` to `EndRow`, it will make more sense.

Reply:
I think `logs` is a fine name. Thanks for the explanation.